cli/types/cypress.d.ts

```diff
@@ -5131,6 +5137,21 @@ declare namespace Cypress {
     tag?: string
   }
 
+  interface DevServerOptions {
+    specs: Cypress.Cypress['spec'][]
+    config: {
+      supportFile: string
```

Comment:
Is this optional? (not sure)

Reply:
This interface is what is returned by cypress to the user. so I guess it is not.

Reply:
spoiler alert: it was...